src/swagger2.0.json

```diff
@@ -58,9 +58,28 @@
             ]
         },
         "/klantcontacten": {
+            "get": {
+                "operationId": "klantcontact_list",
+                "description": "Geef een lijst van KLANTCONTACTen.",
+                "parameters": [],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/KlantContact"
+                            }
+                        }
+                    }
+                },
+                "tags": [
+                    "klantcontacten"
+                ]
+            },
             "post": {
                 "operationId": "klantcontact_create",
-                "description": "Registreer een klantcontact bij een ZAAK.\n\nIndien geen identificatie gegeven is, dan wordt deze automatisch gegenereerd.",
+                "description": "Registreer een klantcontact bij een ZAAK.\n\nIndien geen identificatie gegeven is, dan wordt deze automatisch\ngegenereerd.",
```

Comment:
Is dit bewust?

Reply:
ja, in de zin dat het heel convenient is voor tests en wel aansluit bij de zaakidentificatie die ook gegenereerd (kan) word(t)(en)

Reply:
Ik doelde op de \n  ;)

Reply:
Oh :-)

Is een bij-effect van deze beschrijvingen in docstrings te zetten (en dat is weer om de codebase netjes te houden). Na de markdown-conversie slag zijn die eruit (wat redoc etc. doen).

Reply:
Lijkt me geen probleem. De OAS-spec is toch bijna code en het gaat er inderdaad uit bij het renderen van documentatie.